src/docs/product/alerts-notifications/alert-settings.mdx

```diff
@@ -0,0 +1,34 @@
+---
+title: Alert Settings
+sidebar_order: 3
+---
+
+Configure alerts to your team's needs with alert rules and settings. Create new alerts, view alerts, and designate the recipient(s) of each alert.
+
+## Rules
+
+Alert Rules shows all of your project's alerts, their type, name, triggers, and actions. Use this page to create new alerts, determine if a component of your project needs alerting, or recognize unnecessary alerts.
+
+![Alert rules are divided by projects, their conditions/triggers, and actions(s).](alert-listing.png)
+
+## Settings
+
+Alerts have default settings and options to customize alerts to your personal needs.
```

Comment:
@adhiraj Are there more details that we could add here that would be helpful to the customer?